backend/services/EventHandler.py

```
@@ -30,23 +30,17 @@
-    """Handle event image upload and return the filename."""
-    if not image:
-        return None
-    
-    filename = generate_secure_filename(user_id, ".jpg")
-    save_upload_file(image, upload_dir, filename)
-    return filename
+
 
 async def handle_event_upload(
     media_file: Optional[UploadFile],
     folder_name: str
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)